### PR TITLE
April 2024 release blog post

### PR DESCRIPTION
**Pull request by carlobeltrame**
For https://github.com/ecamp/ecamp3/pull/4925

### CODE REVIEW

**Review by BacLuc — APPROVED** [review]

File: src/content/posts/2024-04-release.de.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Bereits drei Wochen nach der letzten Aktualisierung schalten wir schon wieder ein Update auf eCamp v3 auf. Diesmal sind folgende Features dabei:
+
+
+<div class="simple-columns bg-slate-100 dark:bg-slate-800">
+
+![Einladungen im UI akzeptieren](~/assets/images/personal-invitations-de.png)
+
+<div>
+
+Lagereinladungen können neu auch direkt auf der eCamp v3 Webseite akzeptiert werden, vorausgesetzt es wurde die E-Mailadresse eingeladen, welche zu deinem Account gehört. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink} [#4932](https://github.com/ecamp/ecamp3/issues/4932){.issuelink}
```
Comment (simfeld) [inline src/content/posts/2024-04-release.de.md]:
```suggestion
Lagereinladungen können neu auch direkt in eCamp v3 akzeptiert werden, vorausgesetzt die Einladung wurde an die E-Mailadresse deines Accounts geschickt. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink} [#4932](https://github.com/ecamp/ecamp3/issues/4932){.issuelink}
```

File: src/content/posts/2024-04-release.en.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Barely three weeks have passed since the last update of eCamp v3, and already we bring you the next one. This time, the following features are included:
+
+
+<div class="simple-columns bg-slate-100 dark:bg-slate-800">
+
+![Accept invitations in the UI](~/assets/images/personal-invitations-en.png)
+
+<div>
+
+Invitations to camps can now also be accepted directly on the eCamp v3 application. For this to work, your camp leaders need to invite the exact email address belonging to your user account. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink}
```
Comment (simfeld) [inline src/content/posts/2024-04-release.en.md]:
```suggestion
Invitations to camps can now also be accepted directly in eCamp v3, given the invitation was sent to the email address of your user account. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink}
```

**Comment by carlobeltrame** [comment]
@manuelmeister I am done here. It's just missing a title image, otherwise everything should be complete. Feel free to merge once you are satisfied.

**Review by manuelmeister — APPROVED** [review]
Added image and grammar fixes. LGTM

**Pull request merged**